app/routes/application.js

```diff
@@ -8,8 +8,17 @@ const {
 } = Ember;
 
 export default Route.extend({
+  logger: service(),
   smt: service(),
 
+  beforeModel() {
+    this._super(...arguments);
+    // See a list of allowed types in logger.js
+    // Add or remove all your log types here:
+    // this.get('logger').addToLogs('message');
+    // this.get('logger').removeFromLogs('join');
```

Comment:
What do you think about adding a line that would remove all logs at once, when uncommenting it?